types/webpack-dev-server/index.d.ts

```diff
@@ -108,20 +182,11 @@ declare namespace WebpackDevServer {
         /**
          * Specify a host to use. By default this is localhost.
          */
-        host?: string | undefined;
+        host: string;
```

Comment:
```suggestion
        host?: string | undefined;
```
how otherwise it could be defaulting to `localhost`? Also:
https://github.com/webpack/webpack-dev-server/blob/master/lib/Server.js#L1381